### PR TITLE
Specific statuses + reduced writing to store when finding jobs

### PR DESCRIPTION
**Pull request by jpslav**

### CODE REVIEW

**Comment by jpslav** [comment]
@karenc if you agree with my discussion in the #42, then I think this PR can be merged (if you can review).  I started https://github.com/openstax/tutor-js/pull/825 to handle the state updates, and we might also need a tutor-server PR where we check for incomplete jobs (as incomplete no longer includes failed jobs)

File: lev.gemspec [hunk 1]
```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "sqlite3"
   spec.add_development_dependency "debugger"
   spec.add_development_dependency "activejob"
+  spec.add_development_dependency "debugger"
```
Comment (karenc) [inline lev.gemspec]: I'm going to remove this line because it's the same as line 31.

**Comment by karenc** [comment]
Opened a new pull request for this: #43

**Comment by karenc** [comment]
(this can be closed but I can't close it)

**Pull request closed without merging**